### PR TITLE
fix: line type null data labels

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* Line type chart null data dataLabel exposed properly

#### before
image

#### after
image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jungeun-cho — APPROVED** [review]
area, line 둘다 확인했습니다. 
line 좌표 차트 null 처리는 고려안되어있네요. 음...

File: src/js/plugins/raphaelAreaChart.js [hunk 1]
```diff
@@ -432,7 +432,9 @@ class RaphaelAreaChart extends RaphaelLineBase {
     }
 
     dots.forEach(item => {
-      item.endDot.dot.toFront();
+      if (item.endDot) {
+        item.endDot.dot.toFront();
```
Comment (shiren) [inline src/js/plugins/raphaelAreaChart.js:436]: item.endDot?.dot.toFront();

이렇게 할 수 있나요?

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

File: src/js/plugins/raphaelAreaChart.js (hunk 1 above)
Comment (jung-han) [reply src/js/plugins/raphaelAreaChart.js:436]: 린트랑 다른 룰들 업데이트 해야 사용할 수 있을 것 같습니다 ㅎㅎ

**Pull request merged**